<launch>

<!-- wpv4 core-->
<node pkg="wpv4_bringup" type="wpv4_core" name="wpv4_core" output="screen">
    <param name="serial_port" type="string" value="/dev/wpv4_base"/>  
    <param name="imu_odom" type="bool" value="true"/> 
</node>

<!-- wpv4 imu-->
<node pkg="wpv4_bringup" type="wpv4_imu" name="wpv4_imu" output="screen">
    <param name="serial_port" type="string" value="/dev/wpv4_imu"/> 
</node>

<!-- joy node -->
 <node respawn="true" pkg="joy"
       type="joy_node" name="wpv4_joy" >
   <param name="dev" type="string" value="/dev/input/js0" />
   <param name="deadzone" value="0.12" />
 </node>

<!-- Axes Velcmd -->
 <param name="axis_linear" value="1" type="int"/>
 <param name="axis_angular" value="0" type="int"/>
 <param name="scale_linear" value="2" type="double"/>
 <param name="scale_angular" value="2" type="double"/>
 <node pkg="wpv4_bringup" type="wpv4_js_velcmd" name="teleop"/>

</launch>
